test/controllers/EvidenceUploadController.test.ts

```diff
@@ -98,6 +100,18 @@ describe('EvidenceUploadController', () => {
                         .and.to.contain('another-file.jpeg');
                 });
         });
+
+        it('should return 200 and render download links when attachments in session', async () => {
```

Comment:
Scenario tested by this test is same as scenario tested by one above. I suggest merging the two together so one test checks both that filename is rendered on the screen and that it is linked to correct file download path.